src/main/java/com/trixtaro/trixtalogin/login/LoginTimer.java

```diff
@@ -20,7 +20,7 @@ public void run() {
         
         try{
             
-            Thread.sleep(wait_time * 1000);
+            Thread.sleep(wait_time * 9000);
```

Comment:
You dont need modify this constant, you can change "wait_time" in **Trixtalogin.conf**